speech/google/cloud/speech/client.py

```diff
@@ -195,10 +195,11 @@ def sync_recognize(self, sample, language_code=None,
         api_response = self.connection.api_request(
             method='POST', path='speech:syncrecognize', data=data)
 
-        if len(api_response['results']) == 1:
+        results = api_response['results']
+        if len(results) == 1:
+            result = results[0]['alternatives']
```

Comment:
Change it to `results[0]` if you like. This is "clear enough" but doesn't emphasize the "first item" part as I'd hoped. Not a dealbreaker though.